packages/website/plugins/add-rule-attributes-list.ts

```diff
@@ -0,0 +1,203 @@
+import type * as mdast from 'mdast';
+import type { Plugin } from 'unified';
+
+import * as eslintPlugin from '@typescript-eslint/eslint-plugin';
+
+const addRuleAttributesList: Plugin = () => {
+  return (root, file) => {
+    if (file.stem == null) {
+      return;
+    }
+
+    const rule = eslintPlugin.rules[file.stem];
+    if (rule == null) {
+      return;
+    }
+    const config = ((): 'recommended' | 'strict' | null => {
+      switch (rule.meta.docs?.recommended) {
+        case 'error':
+        case 'warn':
+          return 'recommended';
+
+        case 'strict':
+          return 'strict';
+
+        default:
+          return null;
+      }
+    })();
+    const autoFixable = rule.meta.fixable != null;
+    const suggestionFixable = rule.meta.hasSuggestions === true;
+    const requiresTypeInfo = rule.meta.docs?.requiresTypeChecking === true;
+
+    const parent = root as mdast.Parent;
+    /*
```

Comment:
I found the package that does this for us:

```js
import { fromMarkdown } from "mdast-util-from-markdown";

fromMarkdown(`## Attributes
  ...
`);
```

Especially since we're going to automate more of this page, I feel like that's a good approach?